<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormDeviserLanguage</class>
 <widget class="QWidget" name="FormDeviserLanguage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QGroupBox" name="grpLanguageVersions">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="title">
      <string> Language Versions: </string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QToolButton" name="cmdAddVersion">
          <property name="text">
           <string>&amp;Add</string>
          </property>
          <property name="icon">
           <iconset resource="../deviser-resources.qrc">
            <normaloff>:/images/edit_add.png</normaloff>:/images/edit_add.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="cmdDelVersion">
          <property name="text">
           <string>&amp;Del</string>
          </property>
          <property name="icon">
           <iconset resource="../deviser-resources.qrc">
            <normaloff>:/images/list_remove.png</normaloff>:/images/list_remove.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="tblLanguageVersions">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>23</number>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="grpDependencies">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="title">
      <string> Dependency Libraries: </string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QToolButton" name="cmdAddDependency">
          <property name="text">
           <string>&amp;Add</string>
          </property>
          <property name="icon">
           <iconset resource="../deviser-resources.qrc">
            <normaloff>:/images/edit_add.png</normaloff>:/images/edit_add.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="cmdDelDependency">
          <property name="text">
           <string>&amp;Del</string>
          </property>
          <property name="icon">
           <iconset resource="../deviser-resources.qrc">
            <normaloff>:/images/list_remove.png</normaloff>:/images/list_remove.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="tblDependencies">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>23</number>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="chkIsPackage">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Is Package</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="grpLanguage" native="true">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Markup Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txtName">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The name of the language, that the library is implementing.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Prefix:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="txtPrefix">
        <property name="toolTip">
         <string/>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Document Class:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="txtDocumentClass">
        <property name="toolTip">
         <string/>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Base Class:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="txtBaseClass">
        <property name="toolTip">
         <string/>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Library Name:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="txtLibraryName">
        <property name="toolTip">
         <string/>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>List of Class:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="txtListOfName">
        <property name="toolTip">
         <string/>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Annotation Element Name:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="txtAnnotationElementName">
        <property name="toolTip">
         <string/>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Major: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtMajor">
            <property name="toolTip">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Minor: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtMinor">
            <property name="toolTip">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_21">
          <item>
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>Revision: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtRevision">
            <property name="toolTip">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Library Version</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../deviser-resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cmdAddDependency</sender>
   <signal>clicked()</signal>
   <receiver>FormDeviserLanguage</receiver>
   <slot>addDependency()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>31</x>
     <y>372</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkIsPackage</sender>
   <signal>clicked()</signal>
   <receiver>FormDeviserLanguage</receiver>
   <slot>isPackageToggled()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdAddVersion</sender>
   <signal>clicked()</signal>
   <receiver>FormDeviserLanguage</receiver>
   <slot>addVersion()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>31</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdDelDependency</sender>
   <signal>clicked()</signal>
   <receiver>FormDeviserLanguage</receiver>
   <slot>delDependency()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>31</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdDelVersion</sender>
   <signal>clicked()</signal>
   <receiver>FormDeviserLanguage</receiver>
   <slot>delVersion()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>31</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtBaseClass</sender>
   <signal>textEdited(QString)</signal>
   <receiver>FormDeviserLanguage</receiver>
   <slot>baseClassChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtDocumentClass</sender>
   <signal>textEdited(QString)</signal>
   <receiver>FormDeviserLanguage</receiver>
   <slot>documentClassChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtLibraryName</sender>
   <signal>textEdited(QString)</signal>
   <receiver>FormDeviserLanguage</receiver>
   <slot>libraryNameChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtMajor</sender>
   <signal>textEdited(QString)</signal>
   <receiver>FormDeviserLanguage</receiver>
   <slot>majorChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtMinor</sender>
   <signal>textEdited(QString)</signal>
   <receiver>FormDeviserLanguage</receiver>
   <slot>minorChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtName</sender>
   <signal>textEdited(QString)</signal>
   <receiver>FormDeviserLanguage</receiver>
   <slot>markupNameChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtPrefix</sender>
   <signal>textEdited(QString)</signal>
   <receiver>FormDeviserLanguage</receiver>
   <slot>prefixChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtRevision</sender>
   <signal>textEdited(QString)</signal>
   <receiver>FormDeviserLanguage</receiver>
   <slot>revisionChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>564</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtAnnotationElementName</sender>
   <signal>textEdited(QString)</signal>
   <receiver>FormDeviserLanguage</receiver>
   <slot>annotationElementNameChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>386</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtListOfName</sender>
   <signal>textEdited(QString)</signal>
   <receiver>FormDeviserLanguage</receiver>
   <slot>listOfNameChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>386</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>isPackageToggled()</slot>
  <slot>markupNameChanged(QString)</slot>
  <slot>prefixChanged(QString)</slot>
  <slot>documentClassChanged(QString)</slot>
  <slot>baseClassChanged(QString)</slot>
  <slot>libraryNameChanged(QString)</slot>
  <slot>listOfNameChanged(QString)</slot>
  <slot>annotationElementNameChanged(QString)</slot>
  <slot>majorChanged(QString)</slot>
  <slot>minorChanged(QString)</slot>
  <slot>revisionChanged(QString)</slot>
  <slot>addVersion()</slot>
  <slot>delVersion()</slot>
  <slot>addDependency()</slot>
  <slot>delDependency()</slot>
 </slots>
</ui>
